app/src/test/java/decide/LIC3Test.java

```diff
@@ -0,0 +1,94 @@
+package decide;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+import decide.datastructures.Parameters;
+import decide.datastructures.Point;
+import decide.lic.LIC3;
+
+public class LIC3Test {
+
+    LIC3 lic3 = new LIC3();
+    Parameters parameters = new Parameters();
```

Comment:
Having static instances is probably not a good way of doing it, in more complicated test cases you could have several different parameters that need to be changed, and if one is forgotten in one test case then previous data from a previous test case is used, which could lead to weird behavior. (The test case passes on its own, but not when running several tests at once for example).

I would suggest a new instance inside each test case